client/src/lib.rs

```diff
@@ -311,108 +489,3 @@ impl<C: ChainSpec + 'static> SubxtClientConfig<C> {
         service_config
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
removed these tests because of https://github.com/paritytech/substrate-subxt/pull/236#issuecomment-791976802